src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +333,27 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static ItemExecutor getItemManager() {
+        return MythicBukkit.inst().getItemManager();
+    }
+
+    public static Collection<String> getItemNames() {
+        return getItemManager().getItemNames();
+    }
+
+    public static MythicItem getMythicItem(String name) {
```

Comment:
Same here, only used in one place and isn't a lot cleaner then using `getItemManager().getItem(name)` directly